xwiki-platform-tools/xwiki-platform-tool-packager-plugin/src/main/java/org/xwiki/tool/packager/PackageMojo.java

```diff
@@ -351,6 +351,21 @@ private Artifact resolveArtifactItem(ArtifactItem artifactItem) throws MojoExecu
         return artifact;
     }
 
+    /**
+     * Resolve {@code xwiki-platform-tool-configuration-resources}, then look for velocity scripts inside the artifact
+     * and interpret them before copying the result to {@code configurationFileTargetDirectory}.
+     * <p>
+     * <a 
+     *   href="https://sonarcloud.io/organizations/xwiki/rules?open=javasecurity%3AS6096&rule_key=javasecurity%3AS6096">
+     *   javasecurity:S6096
+     * </a> is ignored because we trust the content of {@code xwiki-platform-tool-configuration-resources} since it is 
+     * produced by maven using source code we own.
+     *
+     * @param configurationFileTargetDirectory the root directory where the configuration files found in {@code
+     *     xwiki-platform-tool-configuration-resources} are copied ({@code "webapps/xwiki/WEB-INF"} by default)
```

Comment:
Note: We shouldn't put quotes inside `@code` annotation

Reply:
Ok, what I meant by comment was simpler (no need for javadoc especially since it's an impl detail) ;) It was simply a comment where we extract the zip to say that we don't need to verify if there's no relative path to prevent against zip slip attacks since we control the JAR source.

Reply:
Thanks :) I'll add an inline comment too, I feel like the "zip slip" keyword is missing somewhere.